rush.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
+
+  "rushVersion": "5.1.0",
+  "pnpmVersion": "2.15.1",
+  "nodeSupportedVersionRange": ">=6.9.0 <7.0.0 || >=8.9.4 <9.0.0",
+
+  "projectFolderMinDepth": 2,
+
+  "approvedPackagesPolicy": {
+    "reviewCategories": [ "main", "tools", "prototypes" ],
+    "ignoredNpmScopes": [ "@types" ]
+  },
+
+  "gitPolicy": {
+    // Require GitHub scrubbed e-mails
+    "allowedEmailRegExps": [
+      "[^@]+@users\\.noreply\\.github\\.com"
+    ],
+    "sampleEmail": "mrexample@users.noreply.github.com"
+  },
+
+  "projects": [
+    {
+      "packageName": "my-app",
+      "projectFolder": "apps/my-app",
+      "reviewCategory": "main"
+    },
+    {
+      "packageName": "my-controls",
+      "projectFolder": "libraries/my-controls",
+      "reviewCategory": "main"
+    },
+    {
+      "packageName": "my-toolchain",
+      "projectFolder": "tools/my-toolchain",
+      "reviewCategory": "tools"
+    }
```

Comment:
add publishing stuff in a comment

Reply:
I'll do that in the next PR.

---
In reply to: [216845660](https://github.com/Microsoft/rush-example/pull/1#discussion_r216845660) [](ancestors = 216845660)